### PR TITLE
Update using_helm.md

### PR DESCRIPTION
**Pull request by pmcanseco**
In docs/using_helm.md, it says on line 214 that you can pass in a YAML formatted file, but the example commands following that sentence create JSON code but names the file with a .yml extension. For clarity, I propose saying that it will accept JSON or YAML but clarify in the code that for the example we're making a JSON file.

### CODE REVIEW

**Comment by mattfarina** [comment]
@pmcanseco Thanks for the contribution. On [Helm we use a DCO](https://www.helm.sh/blog/helm-dco/index.html). Could you please provide signoff on your commit. Directions are on the checks tab of the PR.

**Comment by pmcanseco** [comment]
> @pmcanseco Thanks for the contribution. On [Helm we use a DCO](https://www.helm.sh/blog/helm-dco/index.html). Could you please provide signoff on your commit. Directions are on the checks tab of the PR.

done!

**Comment by bacongobbler** [comment]
`-f` only accepts YAML as input so the docs are wrong in this case. Can we instead make the output of the `echo` properly-formed YAML? https://github.com/helm/helm/blob/1801fa0074a0207f4d5724667858b53d52c486f8/cmd/helm/install.go#L197

**Comment by bacongobbler** [comment]
ping

**Comment by bacongobbler** [comment]
It looks like the DCO check failed again. Would you mind amending your last commit and sign off on it? Thanks again.

**Comment by pmcanseco** [comment]
I've amended my commit and it seems to be good now. Thanks for your patience.

File: docs/using_helm.md [hunk 1]
```diff
@@ -211,11 +211,14 @@ imageTag: 10.1.14-r3
 # mariadbDatabase:
 ```
 
-You can then override any of these settings in a YAML formatted file,
+You can then override any of these settings in a JSON or YAML formatted file,
```
Comment (bacongobbler) [inline docs/using_helm.md]: "JSON or" should be removed based on previous feedback (-f doesn't handle JSON files)

**Review by bacongobbler — CHANGES_REQUESTED** [review]
One last change. Sorry!

**Comment by pmcanseco** [comment]
Woops, fixed!

**Review by bacongobbler — APPROVED** [review]

**Comment by bacongobbler** [comment]
Thanks!

**Pull request merged**